src/components/cart/cart-item/cart-item-view.tsx

```diff
@@ -0,0 +1,69 @@
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { cart } from '@wix/ecom';
+import classNames from 'classnames';
+import { getImageHttpUrl } from '~/api/wix-image';
+import { Price } from '~/components/price/price';
+import { isCartItemAvailable } from '~/utils';
+import styles from './cart-item-view.module.scss';
+
+export interface CartItemViewProps {
+    className?: string;
+    isLast?: boolean;
+    cartItem: cart.LineItem;
+    onRemoveButtonClick: () => void;
+    onQuantityChange: React.ChangeEventHandler<HTMLInputElement>;
+}
+
+export const CartItemView = ({
+    cartItem,
+    className,
+    isLast,
```

Comment:
this is just moved from `CartItem` component... `isLast` is weird and should be removed in the next PR